doc/man3/BIO_set_callback.pod

```diff
@@ -212,9 +212,22 @@ after.
 The BIO_debug_callback() function is a good example, its source is
 in crypto/bio/bio_cb.c
 
+=head1 RETURN VALUES
+
+BIO_set_callback_ex(), BIO_set_callback() and BIO_set_callback_arg() do not
+return a value.
+
+BIO_get_callback_ex() and BIO_get_callback() return BIO_callback_fn_ex()
+and BIO_callback_fn() respectively.
+
+BIO_get_callback_arg() returns a B<char> pointer.
```

Comment:
To what? Just telling us the type is self evident. More useful is what the data represents, i.e. the value previously set via a call to BIO_set_callback_arg()